tools/ldcontext_generator.py

```diff
@@ -173,6 +209,31 @@ def aggregate_ld_context(f, uri_prefix, predefined_mappings):
     for p in ld_context:
         aggregated_context[p] = ld_context[p]
 
+        if p not in mapping_list['terms']:
```

Comment:
can we put this code that has just to do with the yaml generation in a separate function?

Reply:
Almost done, not sure if in a correct form